REST/RESTListener.cc

```diff
@@ -145,10 +153,21 @@ namespace litecore { namespace REST {
             tlsContext->requirePeerCert(true);
         if (tlsConfig->rootClientCerts.buf)
             tlsContext->setRootCerts(tlsConfig->rootClientCerts);
+        if (auto callback = tlsConfig->certAuthCallback; callback) {
+            auto context = tlsConfig->tlsCallbackContext;
+            tlsContext->setCertAuthCallback([=](slice certData) {
+                return callback((C4Listener*)this, certData, context);
+            });
+        }
         return tlsContext;
     }
 
 
+    unsigned RESTListener::connectionCount() {
+        return _server->connectionCount();
```

Comment:
Why is one of these a signed int and the other an unsigned?  I think we should settle on one or the other.